[2024-09-18 16:04:55,365] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Regions_ETL.envio_mail manual__2024-09-18T16:04:14.994469+00:00 [queued]>
[2024-09-18 16:04:55,407] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Regions_ETL.envio_mail manual__2024-09-18T16:04:14.994469+00:00 [queued]>
[2024-09-18 16:04:55,408] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-09-18 16:04:55,409] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2024-09-18 16:04:55,409] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-09-18 16:04:55,451] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): envio_mail> on 2024-09-18 16:04:14.994469+00:00
[2024-09-18 16:04:55,459] {standard_task_runner.py:52} INFO - Started process 285 to run task
[2024-09-18 16:04:55,467] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Regions_ETL', 'envio_mail', 'manual__2024-09-18T16:04:14.994469+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Regions.py', '--cfg-path', '/tmp/tmp3iqzgqay', '--error-file', '/tmp/tmp1gfsp_t6']
[2024-09-18 16:04:55,471] {standard_task_runner.py:77} INFO - Job 11: Subtask envio_mail
[2024-09-18 16:04:55,624] {logging_mixin.py:109} INFO - Running <TaskInstance: Regions_ETL.envio_mail manual__2024-09-18T16:04:14.994469+00:00 [running]> on host b9afb08ecd2d
[2024-09-18 16:04:55,717] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-09-18 16:04:55,774] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=steikedgar99_coderhouse
AIRFLOW_CTX_DAG_ID=Regions_ETL
AIRFLOW_CTX_TASK_ID=envio_mail
AIRFLOW_CTX_EXECUTION_DATE=2024-09-18T16:04:14.994469+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-18T16:04:14.994469+00:00
[2024-09-18 16:04:55,776] {logging_mixin.py:109} INFO - Enviando Email: 2024-09-18 16
[2024-09-18 16:04:57,886] {logging_mixin.py:109} INFO - (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials d9443c01a7336-207945db101sm67037815ad.1 - gsmtp')
[2024-09-18 16:04:57,886] {logging_mixin.py:109} INFO - El envio ha fallado
[2024-09-18 16:04:57,887] {python.py:175} INFO - Done. Returned value was: None
[2024-09-18 16:04:57,905] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Regions_ETL, task_id=envio_mail, execution_date=20240918T160414, start_date=20240918T160455, end_date=20240918T160457
[2024-09-18 16:04:57,978] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-09-18 16:04:58,088] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
